R/lsmeans.R

```diff
@@ -41,7 +41,7 @@ lsmeans.tern_gee_logistic <- function(object,
   )
 
   prop_df <- cbind(
-    as.data.frame(prop_emm)[, c(object$vars$arm, "prob", "SE", "asymp.LCL", "asymp.UCL")],
+    data.frame(confint(prop_emm))[, c(object$vars$arm, "prob", "SE", "asymp.LCL", "asymp.UCL")],
```

Comment:
```suggestion
    data.frame(stats::confint(prop_emm))[, c(object$vars$arm, "prob", "SE", "asymp.LCL", "asymp.UCL")],
```
to fix the failing CI check

Reply:
Yes, it's true toat `confint.emmGridI()` is a method for `confint`, which is in the `stats` package. I really fail to understand why that would be an issue in any application scenario, because obviously we are doing statistics and you'd be crazy to not have the **stats** package loaded. Shouldn't **tern.gee** require **stats**?

Nevertheless, I pushed up a change as shown.

Reply:
Thanks @rvlenth , this basically comes up as a note in `R CMD check` and we don't import the whole `stats` package to fix this otherwise. Thanks!